source/fuzz/transformation_add_bit_instruction_synonym.h

```diff
@@ -128,6 +128,11 @@ class TransformationAddBitInstructionSynonym : public Transformation {
   static uint32_t GetRequiredFreshIdCount(opt::IRContext* ir_context,
                                           opt::Instruction* bit_instruction);
 
+  // Return true, if instructions operands have the same sign, otherwise,
```

Comment:
Don't forget to adjust the docs. This will be something like
```
Returns true if:
- A |bit_instruction| is one of OpBitwiseOr, OpBitwiseAnd, OpBitwiseXor or OpNot.
- |bit_instruction|'s operands are scalars.
- The operands have the same signedness.
```

Reply:
Done.